<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filebookinsert</class>
 <widget class="QDialog" name="Filebookinsert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪劲楷简</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图书批量入库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪劲楷简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_filename">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪劲楷简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导入图书</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_bookinfo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>651</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_new">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>321</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_old">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>321</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
